src/services/jwt.services.js

```diff
@@ -0,0 +1,47 @@
+const jwt = require('jsonwebtoken');
+const { userDto } = require('../dto/user.dto');
+const { ApiError } = require('../exeptions/auth.error');
+
+const sign = (payload, key, expiresIn) => jwt.sign(payload, key, { expiresIn });
+
+const verify = (payload, key) => {
+  try {
+    return jwt.verify(payload, key);
+  } catch {
+    throw ApiError.unauthorized('The token is invalid');
+  }
+};
```

Comment:
The `verify` function should catch specific errors from `jwt.verify` to provide more detailed error messages. Consider logging the error or differentiating between expired tokens and other verification errors.